commands/shooter/fendershot.py

```diff
@@ -0,0 +1,28 @@
+from commands2 import CommandBase
+from wpimath.geometry import Rotation2d
+
+from subsystems.shootingsubsystem import ShootingSubsystem
+import constants
+
+
+class FenderShot(CommandBase):
+    def __init__(self, shoot: ShootingSubsystem):
+        self.shoot = shoot
+        self.setName(__class__.__name__)
+        self.addRequirements([self.shoot])
+
+    def initialize(self) -> None:
+        print(f"Command: {self.getName()}")
+
+    def execute(self) -> None:
+        self.shoot.rotateTurret(Rotation2d.fromDegrees(0))
+        self.shoot.setHoodAngle(constants.kFenderHoodAngle)
+        self.shoot.setWheelSpeed(constants.kFenderWheelSpeed)
+
```

Comment:
Appears to be missing the actual command to shoot once the turret/hood/flywheel are at the desired positions/velocity

Reply:
fender shot sets the values of the entire system to what a fender shot should be - needs better explanation